app/controllers/base/WizardController.java

```diff
@@ -100,18 +100,17 @@ protected WizardController(HttpExecutionContext ec,
             }
 
         }, ec.current()).
-            exceptionally(throwable -> {
+                handleAsync((result, e) -> Optional.ofNullable(e).map(throwable -> {
+                    Logger.error("Wizard Get failure", throwable);
 
-                Logger.error("Wizard Get failure", throwable);
+                    if (throwable instanceof CompletionException &&
+                            throwable.getCause() instanceof InvalidCredentialsException) {
 
-                if (throwable instanceof CompletionException &&
-                    throwable.getCause() instanceof InvalidCredentialsException) {
+                        return ((InvalidCredentialsException) throwable.getCause()).getErrorResult();
+                    }
 
-                    return ((InvalidCredentialsException) throwable.getCause()).getErrorResult();
-                }
-
-                return internalServerError();
-            });
+                    return internalServerError(renderErrorMessage("We are unable to process your request. Please try again later."));
```

Comment:
Main change is here - instead of return 500 with no body, now returns 500 with a page with a message on.